README.md

```diff
@@ -217,6 +217,16 @@ sh scripts/ibc/setup_ibc.sh
 
 After successful run, the new established channels will be shown
 
+### Set channel-filter
+
+After successful run, set channel-filter for flush to work properly, e.g:
+
+```shell
+src-channel-filter:
```

Comment:
can we please set the channel-filter automatically in the setup_ibc script after successfull ibc_creation? or alternatively give a `sed` command for the user to run manually in this stage?